### ISSUE TITLE
ModuleNotFoundError: No module named 'coc.utils'

### ISSUE THREAD

**Issue by leon1995** [mathsman5133/coc.py#48]
When I try to import `from coc import wars` then I get the following error:
```
$ pyhon3
>>> from coc import wars
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/path/to/python3.7/site-packages/coc/wars.py", line 30, in <module>
    from .iterators import LeagueWarIterator
  File "/path/to/python3.7/site-packages/coc/iterators.py", line 31, in <module>
    from .utils import item
ModuleNotFoundError: No module named 'coc.utils'
```

```
$ pip3 show coc.py
Name: coc.py
Version: 0.3.4
Summary: A python wrapper for the Clash of Clans API
Home-page: https://github.com/mathsman5133/coc.py
Author: mathsman5133
Author-email: None
License: MIT
Location: /path/to/python3.7/site-packages
Requires: aiohttp
Required-by: 
```

Can someone help me with that?

**Comment by mathsman5133** [comment]
Hi, out of curiousity, why are you trying to import the wars module? All the classes within the module should be accessible directly from `__init__.py`, ie. you can do `from coc import ClanWar`. If you have a valid use for importing the module I'll look into it, but that is the preferred way at the moment.

**Comment by leon1995** [comment]
Because when i just import coc then it seems like that coc has no attribute naming WarAttack

**Comment by mathsman5133** [comment]
[It does](https://github.com/mathsman5133/coc.py/blob/7986d0dfa7bd84a541e1cd240ff82af76bb7460c/coc/__init__.py#L79)

You should be able to do `from coc import WarAttack`, or 
```py
import coc

coc_war_attack = coc.WarAttack
```

Responding to your edit: do you have a file or module named `coc`?

**Comment by leon1995** [comment]
It seems like that in the installation folder of coc the `__ini__.py` is missing somehow

**Comment by mathsman5133** [comment]
Strange. Can you have a quick look to see if the `utils.py` file is there either? Otherwise would a fresh install be possible?

**Comment by leon1995** [comment]
utils is missing to. I´ll try a fresh installaion

**Comment by mathsman5133** [comment]
Cool. I suspect this may be the cause of your original problem, too.

**Comment by leon1995** [comment]
Its strange but I can import coc now and it works
```python
import coc

coc_war_attack = coc.WarAttack
```

But when i try to login:
`coc_client = coc.login(email="my_email", password="my_password")` then:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/aiohttp/connector.py", line 967, in _create_direct_connection
    traces=traces), loop=self._loop)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/connector.py", line 830, in _resolve_host
    self._resolver.resolve(host, port, family=self._family)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/resolver.py", line 30, in resolve
    host, port, type=socket.SOCK_STREAM, family=family)
  File "/usr/local/Cellar/python/3.7.8/Frameworks/Python.framework/Versions/3.7/lib/python3.7/asyncio/base_events.py", line 792, in getaddrinfo
    None, getaddr_func, host, port, family, type, proto, flags)
  File "/usr/local/Cellar/python/3.7.8/Frameworks/Python.framework/Versions/3.7/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/Cellar/python/3.7.8/Frameworks/Python.framework/Versions/3.7/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/coc/login.py", line 54, in login
    instance.loop.run_until_complete(instance.login(email, password))
  File "/usr/local/Cellar/python/3.7.8/Frameworks/Python.framework/Versions/3.7/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.7/site-packages/coc/client.py", line 167, in login
    await self.http.get_keys()
  File "/usr/local/lib/python3.7/site-packages/coc/http.py", line 158, in get_keys
    ip_ = await self.get_ip()
  File "/usr/local/lib/python3.7/site-packages/coc/http.py", line 254, in get_ip
    async with self.__session.request("GET", url) as response:
  File "/usr/local/lib/python3.7/site-packages/aiohttp/client.py", line 1012, in __aenter__
    self._resp = await self._coro
  File "/usr/local/lib/python3.7/site-packages/aiohttp/client.py", line 483, in _request
    timeout=real_timeout
  File "/usr/local/lib/python3.7/site-packages/aiohttp/connector.py", line 523, in connect
    proto = await self._create_connection(req, traces, timeout)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/connector.py", line 859, in _create_connection
    req, traces, timeout)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/connector.py", line 971, in _create_direct_connection
    raise ClientConnectorError(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host api.ipify.org:443 ssl:default [nodename nor servname provided, or not known]
```

I know its a different error and I could open a new issue if you want.

**Comment by mathsman5133** [comment]
Nah this issue is fine.

Are you on a restricted network or the like? Or do you have any weird proxies running? Windows Linux or mac? Can you open api.ipify.org in a browser and see if it works? When I wrote this part I do recall having to shop around a bit as a couple others didn't work for everyone, but this one seemed fairly reliable

**Comment by leon1995** [comment]
Ofc my super dooper pi-hole block the domain... Now it works like a charm. Thanks for the super quick help!
